docs/naisjob/README.md

```diff
@@ -24,16 +27,21 @@ spec:
 ```
 
 !!! warning
-    If your Naisjob is running in GCP, or if it is using [Vault sidecar](../naisjob/reference#vaultsidecar) or [Securelogs](../naisjob/reference/#securelogs).
-    You need to shut down all containers running in the pod for the job to complete.
+    If your Naisjob is [running in GCP](../clusters/gcp/), or if it is using [Vault sidecar](reference/#vaultsidecar) or [Securelogs](reference/#securelogs).
+    You need to shut down all containers running in the pod for the `Job` to complete.
     See [Shutting down extra containers](#shutting-down-extra-containers) for more information.
 
 ## Applying your Naisjob to Kubernetes
 You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).
 
 ## Shutting down extra containers
-If your Naisjob is running in GCP, or if it is using [Vault sidecar](../naisjob/reference#vaultsidecar) or [Securelogs](../naisjob/reference/#securelogs).
-You need to shut down all containers running in the pod for the job to complete.
+If your Naisjob has any of these traits:
+
+ * [runs in GCP](../clusters/gcp/)
+ * [uses Vault sidecar](reference/#vaultsidecar)
+ * [uses Securelogs](reference/#securelogs)
+
+You will need to shut down all containers running in the pod for the `Job` to complete.
```

Comment:
Here I feel as if lines 30-31 (in new version) are repeated (almost verbatim) in more verbose (and easier to read) form in lines 38-44.

Here's my suggested edit:
```suggestion
    Make sure to read the caveat [about shutting down all containers](#shutting-down-extra-containers) for your Naisjob to successfully complete.

## Applying your Naisjob to Kubernetes

You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).

## Shutting down extra containers

If your Naisjob has any of these traits:

     * [runs in GCP](../clusters/gcp/)
     * [uses Vault sidecar](reference/#vaultsidecar)
     * [uses Securelogs](reference/#securelogs)

You will need to shut down all containers running in the pod for the `Job` to complete.
```